### ISSUE TITLE
Updates to Croatian translation and pipeline

### ISSUE THREAD

**Issue by tlacoyodefrijol** [openspending/openspending#1458]
Boris sent an update of how the Croatian locale translation ("HR") should look like with the new datasets. There is also a change in the pipelines. Both lists are below. 

- In the vertical bar, "Izvršenje" instead of "Izvršeno"
- In the vertical bar, "Organizacijska klasifikacija" instead of "prva razina organizacijske klasifikacije(Razdjel)"
- In the vertical bar, "Izvori financiranja" instead of "Izvor financiranja"
- everywhere, "mlrd" instead of Miliarda

Changes in Dataset:

There will be no column "Izvorni plan", just 2 columns ("Tekući plan" and "Izvršenje")
The column name "Izvršeno" changes to "Izvršenje"
"Izvor financiranja" will be on first level aggregate instead of the second level as it is now, but this shouldn't change the structure of the dataset. There will be fewer rows, however.
"#" as the content of some rows and columns will be changed into "Ostalo" or some similar text (this is not changing the structure of dataset)
The short version of the text (all columns starting with "Naziv") will be replaced with a longer version of the text from SAP, also without dataset structure change

**Comment by nightsh** [comment]
@tlacoyodefrijol will the new dataset be remote, or do we have to add it to the repo as well?
